services/app/pages/settings/AccountDeletion.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="flex flex-col md:flex-row mt-4 ml-10 md:ml-4">
+    <div class="w-full md:w-1/4 mb-4 md:mb-0">
+      <h1 class="font-bold text-2xl mb-8 md:mb-14">Account Settings</h1>
+
+      <div class="flex flex-col items-center border max-w-xs p-6">
+        <div class="w-20">
+          <img src="/avatar31.png" alt="Avatar">
+        </div>
+        <p class="text-sky-400 font-bold">Jay Rutherford</p>
+        <p class="text-xs mb-4">Professional title</p>
+        <p class="text-xs text-gray-400 text-center mb-4">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+        <hr class="border-gray-200 w-full mb-4">
+        <div class="flex flex-col items-start">
+          <p class="font-semibold text-xs mb-1">Profile link</p>
+          <div class="flex items-center bg-gray-100 px-2 py-1">
+            <a href="#" class="text-xs font-extralight font-semibold mr-4">https://www.profilelink.com</a>
+            <img class="w-4" src="/copypaste.png" alt="copy paste">  
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="ml-6 md:ml-8 w-3/4 md:mr-8 mt-2">
+  
+      <div class="mb-8 md:mb-14 flex flex-nowrap md:ml-2">
+        <a href="#" @click.prevent="setActive('General')"
+        :class="{'text-blue-500 border-b-2 border-blue-500': activeLink === 'General', 'text-gray-500': activeLink !== 'General'}" class="text-sm md:ml-8 mr-4 md:mr-24 text-sm text-gray-500 font-semibold">General</a>
+
+        <a href="#" @click.prevent="setActive('Security')"
+        :class="{'text-blue-500 border-b-2 border-blue-500': activeLink === 'Security', 'text-gray-500': activeLink !== 'Security'}" class="text-sm mr-4 md:mr-24 text-sm text-gray-500 font-semibold">Security</a>
+
+        <a href="#" @click.prevent="setActive('Notifications')"
+        :class="{'text-blue-500 border-b-2 border-blue-500': activeLink === 'Notifications', 'text-gray-500': activeLink !== 'Notifications'}" class="text-sm mr-4 md:mr-24 text-sm text-gray-500 font-semibold">Notifications</a>
+
+        <a href="#" @click.prevent="setActive('Account Deletion')"
+        :class="{'text-blue-500 border-b-2 border-blue-500': activeLink === 'Account Deletion', 'text-gray-500': activeLink !== 'Account Deletion'}" class="text-sm md:mr-24 text-sm text-gray-500 font-semibold">Account Deletion</a>
+      </div>
+
```

Comment:
Currently, there are four repetitive blocks of code above, all using @click.prevent for different tabs (General, Security, etc.). Instead of hardcoding each block, you can turn this into a dynamic loop that will render based on an array. This way, you're reducing redundancy and making the code easier to maintain.